core/src/toga/platform.py

```diff
@@ -20,7 +20,8 @@
     "android": "android",
     "darwin": "macOS",
     "ios": "iOS",
-    "linux": "linux",
+    "linux": "UnixLike",
+    "freebsd13": "UnixLike",
```

Comment:
This breaks backwards compatibility for existing code that is expecting "linux" as the toga platform.

The right approach here is to add a mapping for `freebsd13` to `freebsd` (or, to `freebsd13` if you think there's a need to differentiate versions in a Toga context; I can't say I know FreeBSD well enough to comment on whether this is the case). Adding mappings for 12 and 14 would also make sense.

Reply:
The changes addressed into the following commits, with removing the `freebsd` version.